config/routes.rb

```diff
@@ -55,6 +55,7 @@ def set_up_flipper
     resources :partners, except: %i[new create destroy]
     resources :users
     resources :barcode_items
+    resources :account_requests
```

Comment:
It looks like you're only adding an index route right? I generally prefer doing `only: [:index]` if so, but I'm not sure how consistent we've been about that in diaper